playwright.config.ts

```diff
@@ -21,6 +21,7 @@ const config: PlaywrightTestConfig = {
     },
   ],
   testDir: "e2e",
+  // eslint-disable-next-line sonarjs/slow-regex -- input is just local filenames so trusted
   testMatch: /.*\.spec\.ts/,
```

Comment:
Probably preferable to change this to `/\.spec\.ts$/` and not disable the ESLint rule

Reply:
Thanks for catching!